mfr/server/handlers/export.py

```diff
@@ -59,20 +55,27 @@ def get(self):
         )
 
         loop = asyncio.get_event_loop()
-        yield from loop.run_in_executor(None, exporter.export)
+        await loop.run_in_executor(None, exporter.export)
 
         with open(self.output_file_path.full_path, 'rb') as fp:
             self._set_headers()
-            yield self.write_stream(waterbutler.core.streams.FileStreamReader(fp))
+            await self.write_stream(waterbutler.core.streams.FileStreamReader(fp))
 
-    @tornado.gen.coroutine
     def on_finish(self):
         if self.request.method not in self.ALLOWED_METHODS:
             return
 
+        # Spin off upload into non-blocking operation
+        loop = asyncio.get_event_loop()
+        loop.call_soon(
```

Comment:
I did that first, but noticed that render.py [is doing it this way](https://github.com/CenterForOpenScience/modular-file-renderer/blob/feature/python-3.5/mfr/server/handlers/render.py#L61).  Is there a reason for that?